CHANGELOG.md

```diff
@@ -26,6 +31,7 @@ file key4.db and secured with a possibly set primary password.
 
 ### Android
 - Upgraded Kotlin compiler from 1.9.24 to 2.1.20 ([#6640](https://github.com/mozilla/application-services/pull/6640))/([#6654](https://github.com/mozilla/application-services/pull/6654))
+- Upgraded the JNA dependency version to 5.17.0. ([#6649](https://github.com/mozilla/application-services/pull/6649))
```

Comment:
I assume these changes are because those other PRs didn't update the changelog?

Reply:
We updated the changelog, backed it out, and are now re-updating it again.

Reply:
@bendk It should be in the 139 section, not 138, however.

Reply:
Nice catch.  I updated the commit to do that.